lib/salus/scanners/osv/maven_osv.rb

```diff
@@ -0,0 +1,150 @@
+require 'salus/scanners/osv/base'
+
+module Salus::Scanners::OSV
+  class MavenOSV < Base
+    class SemVersion < Gem::Version; end
+
+    EMPTY_STRING = "Not Found".freeze
+    DEFAULT_SOURCE = "https://osv.dev/list".freeze
+    DEFAULT_SEVERITY = "MODERATE".freeze
+    GITHUB_DATABASE_STRING = "Github Advisory Database".freeze
```

Comment:
Are there any other scanners that use `GITHUB_DATABASE_STRING`?  If so, maybe move it to a super class?

Reply:
The super class adds a `database` field depending on ID and the sub class cherry picks `GITHUB_DATABASE_STRING`. The reason I added in sub class is to allow sub class to pick advisories they want and the super class returns everything.